### ISSUE TITLE
Search window style broken

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#11804]
### Describe your problem!

On my Android 11 device the style is broken since recent changes:

- Big black frame around window (happens on white and black theme) (Looks same as #11190)
- Interaction with keyboard is strange. Once keyboard is closed, the results vanish as well 
- Text input field blocked by the result window

### How to reproduce?

See video to understand the problems:
https://files.droescher.eu/public/20211008_193305.mp4

### Reproducible

Yes

### c:geo Version

2021.10.08-NB

### System information

```text
Samsung S20
Android 11
```

**Comment by Lineflyer** [comment]
Black frame is solved with recent nightly build 2021.10.09-NB1

Only drawback seems to be that the shadow is now missing as also on e.g. the cache menu.
![Screenshot_20211009-154828_cgeo](https://user-images.githubusercontent.com/949669/136660870-7be5450c-1816-4c23-9746-5b190aef502c.jpg)

**Comment by fm-sys** [comment]
Somehow I really don't like Samsung smartphones. Why do they need to patch their android versions that much, that developers have to spend hours of extra work to get things running on their devices as well?

Anyway, I have absolutely no idea what the problem could be. But let's continue this in  #11144

**Comment by moving-bits** [comment]
Since we found a solution for the shadow - anything left that needs to be done here?

**Comment by fm-sys** [comment]
There was the feature request to don't hide the suggestions if the keyboard gets closed. I've created a PR for it (https://github.com/cgeo/cgeo/pull/11814) but I'm not sure if we really want this behavior... Maybe @Lineflyer can test in the emulator, whether the PR behaves like requested? 

Except this, everything should be solved.

**Comment by fm-sys** [comment]
If extracted the keyboard interaction topic to a new issue (#11833) as it is in fact a feature request and not a regression. So let's close here...
